webpack/assets/javascripts/react_app/components/toast/ToastList.scss

```diff
@@ -0,0 +1,11 @@
+.toast-notifications-list-pf {
```

Comment:
the close button needs some margin from the text, and has annoying outline when clicking.